scripts/sanity_chk/sanitylib.py

```diff
@@ -768,14 +768,17 @@ def _thread(handler, timeout, outdir, logfile, fifo_fn, pid_fn, results, harness
         while True:
             this_timeout = int((timeout_time - time.time()) * 1000)
             if this_timeout < 0 or not p.poll(this_timeout):
-                if pid and this_timeout > 0:
-                    #there is possibility we polled nothing because
-                    #of host not scheduled QEMU process enough CPU
-                    #time during p.poll(this_timeout)
-                    cpu_time = QEMUHandler._get_cpu_time(pid)
-                    if cpu_time < timeout and not out_state:
-                        timeout_time = time.time() + (timeout - cpu_time)
-                        continue
+                try:
+                    if pid and this_timeout > 0:
+                        #there's possibility we polled nothing because
+                        #of not enough CPU time scheduled by host for
+                        #QEMU process during p.poll(this_timeout)
+                        cpu_time = QEMUHandler._get_cpu_time(pid)
+                        if cpu_time < timeout and not out_state:
+                            timeout_time = time.time() + (timeout - cpu_time)
+                            continue
+                except Exception:
```

Comment:
Catching Exception like this is generally a bad practice; is there any more specific exception type that can be caught here instead?

Reply:
@wentongwu  are you going to address this?